apps/back/templates/expense/list.html.twig

```diff
@@ -5,5 +5,33 @@
 {% endblock %}
 
 {% block body %}
-    <h1>Liste des dépenses</h1>
+    <h2>Groupe : {{ group.name }}</h2>
+    <h2>Liste des dépenses</h2>
+    <div>
+        <table>
+            <tr>
+                <th>Nom Payé par</th>
+                <th>Date</th>
+                <th>Montant</th>
+                <th>Participants</th>
+            </tr>
+            {% if group.expensesOfTheGroup is null %}
+                <p>Ce groupe n'a pas encore de dépense.</p>
+            {% else %}
+                {% for expense in group.expensesOfTheGroup|sort((b,a) => a.date <=> b.date) %}
+                <tr>
+                    <td>{{ expense.name }} {{ expense.payer.name }}</td>
+                    <td>{{ expense.date|date('d/m/Y') }}</td>
+                    <td>{{ expense.amount|format_currency }}</td>
+                    <td>{{ expense.participants|length }}</td>
+                </tr>
+                {% endfor %}
+            {% endif %}
```

Comment:
la boucle `for` de twig possede une fonction `else` qui pourrait simplifier le code ici